2022-02-12 04:17:46,558 INFO: logging in using static token
2022-02-12 04:17:46,841 INFO: Shard ID None has sent the IDENTIFY payload.
2022-02-12 04:17:46,954 INFO: Shard ID None has connected to Gateway: ["gateway-prd-main-42dv",{"micros":75112,"calls":["discord-sessions-green-prd-2-145",{"micros":74274,"calls":["start_session",{"micros":35592,"calls":["api-prd-main-xzjp",{"micros":32098,"calls":["get_user",{"micros":4390},"add_authorized_ip",{"micros":1317},"get_guilds",{"micros":8540},"coros_wait",{"micros":2}]}]},"guilds_connect",{"micros":3,"calls":[]},"presence_connect",{"micros":1,"calls":[]}]}]}] (Session ID: a856c901a90b4c2fc23267d14f00ac25).
2022-02-12 04:17:49,032 INFO: 目前登入身份：神奇海螺#3847
2022-02-12 07:41:47,035 INFO: logging in using static token
2022-02-12 07:41:47,285 INFO: Shard ID None has sent the IDENTIFY payload.
2022-02-12 07:41:47,428 INFO: Shard ID None has connected to Gateway: ["gateway-prd-main-ls7q",{"micros":103244,"calls":["discord-sessions-green-prd-2-50",{"micros":102486,"calls":["start_session",{"micros":36450,"calls":["api-prd-main-js4z",{"micros":33122,"calls":["get_user",{"micros":4333},"add_authorized_ip",{"micros":1405},"get_guilds",{"micros":5299},"coros_wait",{"micros":2}]}]},"guilds_connect",{"micros":3,"calls":[]},"presence_connect",{"micros":24460,"calls":[]}]}]}] (Session ID: f4116d868c5ece1123f4e06d13bd01b5).
2022-02-12 07:41:49,499 INFO: 目前登入身份：神奇海螺#3847
2022-02-12 08:21:59,216 INFO: Magic Shell back.
2022-02-12 08:21:59,217 INFO: Connecting to voice...
2022-02-12 08:21:59,217 INFO: Starting voice handshake... (connection attempt 1)
2022-02-12 08:21:59,815 INFO: Voice handshake complete. Endpoint found japan125.discord.media
2022-02-12 08:21:59,955 ERROR: Exception occurred during Voice Protocol voice server update handler
Traceback (most recent call last):
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/state.py", line 102, in logging_coroutine
    await coroutine
  File "/opt/virtualenvs/python3/lib/python3.8/site-packages/discord/voice_client.py", line 295, in on_voice_server_update
    await self.ws.close(4000)
AttributeError: 'NoneType' object has no attribute 'close'
